client/a8c-for-agencies/sections/signup/agency-details-form/hooks/use-create-agency-mutation.ts

```diff
@@ -1,9 +1,13 @@
 import { useMutation, UseMutationOptions, UseMutationResult } from '@tanstack/react-query';
+import cookie from 'cookie';
 import wpcom from 'calypso/lib/wp';
 import { APIError, Agency } from 'calypso/state/a8c-for-agencies/types';
 import { AgencyDetailsPayload } from '../types';
 
 function createAgency( details: AgencyDetailsPayload ): Promise< Agency > {
+	// For Agency signup tracking, we need to submit the hubspotutk cookie.
+	const hubspotutk = cookie.parse( document.cookie ).hubspotutk;
```

Comment:
Let's use the optional chaining here to be on the safer side of property is missing?

```suggestion
	const hubspotutk = cookie.parse( document.cookie )?.hubspotutk;
```